Lib/asyncio/selector_events.py

```diff
@@ -954,6 +1018,38 @@ def _write_ready(self):
                 elif self._eof:
                     self._sock.shutdown(socket.SHUT_WR)
 
+    def _write_vectored_self(self):
+        try:
+            try:
+                n = self._sock.sendmsg(seld._buffer)
```

Comment:
This code typo was likely suppressed because of the `try-except`. It will not work as intended.

```suggestion
                n = self._sock.sendmsg(self._buffer)
```